azurerm/resource_arm_monitor_diagnostic_setting.go

```diff
@@ -73,6 +73,19 @@ func resourceArmMonitorDiagnosticSetting() *schema.Resource {
 				ValidateFunc: azure.ValidateResourceID,
 			},
 
+			"log_analytics_destination_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: false,
+				ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
```

Comment:
I believe this could be:
```suggestion
				ValidateFunc: validation.StringInSlice([]string{"Dedicated"}),
```

Reply:
I've made the change to the ValidateFunc.  The property seems to be case sensitive so I've set ignoreCase to false